src/tests/data/pubkey/sphincsplus_fors.vec

```diff
@@ -0,0 +1,82 @@
4da6e2d627d8f7097b2808fcd852bf03bde7471bdb8cfb1950491bcd098eb27cb018124151c270fb090149c22a67908bb4742d0f055d3c0ebc29e417d2d2cd91a217d5d2677f0c08f12780dabcd9c7296f671c7b9b3d591ed1e4bf6e258b8d5e596cf1327985c4a8e16759749655a6f04645c7c5f4989f97f4430c7d1772a543cf9c8db435e1e5c6373b0ddfa41b2b07cca12a79b7917baf31ebcae4e18e14d27ccca6027166a762e88e3d8ec10b0a014152dcf4c6c6a18eab9854cc007e9649be42a35a20ec822249f4ede3a55a1176ba6a3f461a3a849f8009718d8d1b9fd56813b7f8fbac308f67fac3e5d5c68af2474c3be2d93843f4cbe8162ebcfcd9803bc0de8d55c53c90eced31b531b3c4e66b15bbbbd534dee00999c87a22d66f38dd7b9684d25df4587d7077121482ce007650d8693e95da9071183016bd209b2bcecc0242e052302e4c898ed29c6c011bd2342c8723e1df8634af2b39b5cdc513977c86ee9e1932708e8a08612d9a6faa11590768a9ab38ed24689b3cb5cab23cc4f73fb54a5750c82902728bc7cb1ecb720db4d8574ce4ff68e94abad3f31937a595cc774ee75a5d430cd051f17bc4a0adc65807317b15e331147141a9d4612cf775efba27b825a0bb88514b3543a1f74af30f8007b1d3087b12163add0c2b823adda2bba3fb09ecc6196bf3fdace70909396bc40568044855413320a04790565946d42e110400ee5a88265d60a711cff6b69c939577b5710f89338cdfa9d69d5189b6bff63914e1dd10ea7594e8fc1b07c21f3b68c0c478d7285655e8442b3ae8c248ed5b8846d317d7c4041e8a2bbcf8f401d05eae8a88d77df10140fd36ea872eb3d064837fc410ea3b0fa90a02dbee8ef6b59072f85c0baddebd5a24f07988667245141e98d9c882cf30409682615ef8d598a8569e3aeb9e557b5a5e8e918bfa810d6d750cbe4499bb564384d608cf6ccb7ecff6001011891eab5f522c57c20c0e35d2e545c95477568005e5b499246294a6e34def0728d32acfea91c74b946a5b296e1a6eed55e33a26b72c163ef18021d6517f8413903335d60c4e762b58e9c599ad7412e3e8ba88cd7f32c1e5fd5f5ca65a123746839e0243be0c58829a845057f35b2d227e5692b1b15c0d329624aa004f26c1a3d79e3abd47383c74130051d47dff98a48dabadb3f2bbf01503d1f5cf728e5642f3ac2143da7135dce9875ade0271c08fd542820bf4cfa036334e86a4ac144a466e4e37011633e0b80e21d7550cd0ac61867e348df041e9570614808601c9f43dbaa54b9bb75ce0f309e72f083863a541b31336a042e3d28fd86eda2d7c10f340355f62777bdee01ab8efe9cb340c1ce22eeee93eae324a94e148b45f00243bc1c2e1fcce1ac063c45a270e83be90f3e2d483bca5488b821fe32fcea4841e58c7969273c45ff47f1ad46ba8e791d40106f232959315ee0695e3d83d6fc18b763d37fce98f740bc5e8dcefaa511902227244d3f804814733c56a1763f6e8dd8e3669bf32918e3b7343d2a627e60bb93a455e63331a953b62307163d0d820775faf34ecfa016859d0cb51d379efc3f97145cfab7b5510af9d74ccd866771c9d651da1d4a53a6944ee3e779adee1b2b47311a46a1aeddf91f7ca690d56466871a43bd2897a0aede78876e29350ece1ae3525c23557b2870891e0197cf491ddde0ecfb47a12800710fe87896a23ef8c93dc72ccfed7d5db35157be46d39333a17c457f8bc11af25f9c2bab7f59dec653ea29f370eadd5cfab2e8c9c2f9c8308d9d61e5f0b35b14f7c9ddcb092a57446329f36f6606018a691cb810cdacb39c679ae256dfc5baf2fe7f052e52ccf8638330c0eca69acb447fba35c0b9ca8caaa8a14c8f5e16d7f5b7548ff5dfae726a05749baf6e1f00bbaefccda9f7311ce37c8d81f895c515bc238a2d1bc2158dd37fb699939930dd5343303dc45a5a508846a3d278edab1329b93bc72cada10e6493fe316a3e1f84494382dd1d49e8eb58f5ca291cb3d6c1e5e96d4680f61f4a51d8cc40f688f2cf9636038638b58e62b40bd19b3d9b9f303328cfcef40ce612b5007c3c85b74c3d6173ad8ec1a06d4311e14c68c2ef22b9d
```

Comment:
Here again including only the hashes of the signatures would reduce the amount of test data by quite a log.